grails-app/conf/Config.groovy

```diff
@@ -144,6 +148,7 @@ environments {
         gogoduck.url = "http://${localhostAddress}:8300/go-go-duck"
         geonetwork.url = "http://${localhostAddress}:8883/geonetwork"
         gogoduck.filenamePrepend = "IMOS_aggregation"
+        geonetwork.version = 3
```

Comment:
Do we want to be a bit more specific with the version number in case we have to cater for changes in a subsequent 3.x version?